### PR TITLE
DLP

### PR DESCRIPTION
**Pull request by lukesneeringer**
This adds DLP v2beta1 to google-cloud-python.

### CODE REVIEW

**Comment by lukesneeringer** [comment]
Waiting on CI; I am sure I will have missed something.

File: dlp/setup.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+https://packaging.python.org/en/latest/distributing.html
+https://github.com/pypa/sampleproject
+"""
+
+from setuptools import setup, find_packages
+import io
+
+install_requires = [
+    'google-api-core>=0.1.0, <0.2.0dev',
+    'google-auth>=1.0.2, <2.0dev',
```
Comment (jonparrott) [inline dlp/setup.py]: this should *only* depend on `google-api-core[grpc]`.

**Comment by lukesneeringer** [comment]
@jonparrott Baton to you once CI passes.

**Comment by jonparrott** [comment]
Merge away. I'll do the release in a separate PR.

Let's see if our bot will merge this time.

@dpebot will you merge when tests pass?

**Comment by tseaver** [comment]
@lukesneeringer Is the cryptic `dlp` really the best name we can use here?  I see that it is used on cloud.google.com, so I guess the question is really moot.

**Comment by jonparrott** [comment]
@tseaver yeah, that's going to be the name. I agree three-letter acronyms aren't the best.

**Comment by lukesneeringer** [comment]
Agree with the complaint, and agree that it is not going to change.

**Comment by lukesneeringer** [comment]
It also does not help that the name itself is not great. It is titled "Data Loss Prevention" and it does a bunch of redaction, PII detection, etc.

**Pull request merged**